src/input_device_monitor/my_event/mouse_button.rs

```diff
@@ -0,0 +1,9 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
+pub enum MyMouseButton {
+    MyLeft,
+    MyRight,
+    MyMiddle,
+    MyOther(u16),
```

Comment:
I've noticed there is a lot of My prefixes in this implementation. Is it because there are already names without the prefix?

Reply:
The enums already exist in Nannou, but I need to serialize them to send them through the socket and this is the only way, because I cannot add that property because I cannot change the library code

Reply:
okay, I understand